packages/suite/src/views/wallet/send/components/Options/components/BitcoinOptions/components/CoinControl.tsx

```diff
@@ -105,7 +105,9 @@ export const CoinControl = ({ close }: Props) => {
     const totalOutputs = getTotal(
         outputs.map((_, i) => Number(getDefaultValue(`outputs[${i}].amount`, ''))),
     );
-    const totalOutputsInSats = areSatsUsed ? totalOutputs : totalOutputs * 10 ** network.decimals;
+    const totalOutputsInSats = areSatsUsed
+        ? totalOutputs
+        : Math.round(totalOutputs * 10 ** network.decimals); // round to prevent incorrectly computed decimal, e.g. 0.5988788 produces 59887880.00000001
```

Comment:
please use `formatAmount()` from `wallet-utils`

Reply:
Both changes in 56302c065d02498e4e04b99f2382e9301f2ebf7b.
I think the correct helper to use here is `amountToSatoshi`.